ActionKit.xcodeproj/project.pbxproj

```diff
@@ -401,7 +404,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "ActionKit.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
Can this inference be off? By adding the `@objc` flags, the inference should be fine without this flag, no?

Reply:
The answer was yes, I solved it...thanks @brandons again for the PR! :)